src/cli/serve/serve.test.js

```diff
@@ -81,4 +81,77 @@ describe('applyConfigOverrides', () => {
       },
     });
   });
+
+  it('alters config to enable SAML Mock IdP in serverless dev mode', () => {
+    expect(applyConfigOverrides({}, { dev: true, serverless: true }, {}, {})).toEqual({
+      elasticsearch: {
+        hosts: ['https://localhost:9200'],
+        serviceAccountToken: 'AAEAAWVsYXN0aWMva2liYW5hL2tpYmFuYS1kZXY6VVVVVVVVTEstKiBaNA',
```

Comment:
NIT Would it make sense to try and use the const from [dev_service_account.ts](https://github.com/elastic/kibana/blob/6a7c904f921434fe21dfa00eceabfb5e64e915dc/src/platform/packages/shared/kbn-dev-utils/src/dev_service_account.ts#L16) instead?

Reply:
Yeah, it'd not hurt, let me see if it's as easy as linking to kbn-dev-utils from this test file.